libraries/ghc-prim/GHC/Tuple/Check.hs

```diff
@@ -1,10 +1,11 @@
 {-# LANGUAGE DamlSyntax #-}
 {-# LANGUAGE DataKinds #-}
+{-# LANGUAGE NoImplicitPrelude #-}
 {-# LANGUAGE TypeApplications #-}
 
 module GHC.Tuple.Check where
 
 import GHC.Types
 
 userWrittenTuple : a -> a
-userWrittenTuple = magic @"userWrittenTuple"
+userWrittenTuple = \a -> a
```

Comment:
the definition we use here doesn't really matter, and we can't use `magic` since that's defined in the top level `GHC.Types` (i.e. not `ghc-prim:GHC.Types`)